src/components/Faucet/Hero.tsx

```diff
@@ -95,7 +100,7 @@ export default function Hero({
         className,
       )}
     >
-      {!(isExtensionActive && metaMaskAccount) && <EthIcon />}
+      {!(!showInstallButton && metaMaskAccount) && <EthIcon />}
```

Comment:
Nit (can ignore) all of these boolean and the double not `!(!...)` is a little confusing.
I feel like we could group together and simplify?